### PR TITLE
Feature lwdev 6174 update dependencies after canldes producer task 6001

### PR DESCRIPTION
**Pull request by dmitrysmall**

### CODE REVIEW

File: src/Lykke.Service.CandlesHistory.Services/Candles/RedisCachedCandle.cs [hunk 1]
```diff
@@ -19,8 +19,6 @@ public class RedisCachedCandle
         [Key(6)]
         public double TradingOppositVolume { get; set; }
         [Key(7)]
```
Comment (MikhailGerasimov) [inline src/Lykke.Service.CandlesHistory.Services/Candles/RedisCachedCandle.cs]: Is it safe to delete a field?

File: src/Lykke.Service.CandlesHistory.Services/StartupManager.cs [hunk 2]
```diff
@@ -1,5 +1,4 @@
 ﻿using System.Threading.Tasks;
```
Comment (MikhailGerasimov) [inline src/Lykke.Service.CandlesHistory.Services/StartupManager.cs]: Useless class. Can be deleted

File: src/Lykke.Service.CandlesHistory/Lykke.Service.CandlesHistory.csproj [hunk 3]
```diff
@@ -19,7 +19,7 @@
   <ItemGroup>
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.0" />
     <PackageReference Include="Lykke.AzureStorage" Version="5.0.2" />
-    <PackageReference Include="Lykke.Common" Version="4.0.0" />
+    <PackageReference Include="Lykke.Common" Version="4.6.0" />
```
Comment (MikhailGerasimov) [inline src/Lykke.Service.CandlesHistory/Lykke.Service.CandlesHistory.csproj]: The last version is 5

File: src/Lykke.Service.CandlesHistory.Services/Candles/RedisCachedCandle.cs (hunk 1 above)
Comment (dmitrysmall) [reply src/Lykke.Service.CandlesHistory.Services/Candles/RedisCachedCandle.cs]: Actually, Redis cache is cleaned up and filled with new data at every Lykke.Job.CandlesHistoryWriter restart. We are going to update and restart the *Writer and the reader service of this cache simultaneously, so, they will have similar awaited structure of the cache content.

**Pull request merged**